int32 num_obstacles
CuboidObs[] obs_descs
---
bool status
